### PR TITLE
Adding Timing::equivCells and Timing::makeEquivCells python APIs

### PR DESCRIPTION
**Pull request by bingyuew**
Move makeEquivCells from protected to public in rsz::Resizer and call it in Timing::makeEquivCells. Create Timing::equivCells.

### CODE REVIEW

File: include/ord/Timing.h [hunk 1]
```diff
@@ -63,6 +63,10 @@ class Vertex;
 class Pin;
 }  // namespace sta
 
+namespace rsz {
+  class Resizer;
+}
+
```
Comment (maliberty) [inline include/ord/Timing.h:69]: Why is this needed?  I see no mention of Resizer in this header.

File: src/Timing.cc [hunk 2]
```diff
@@ -385,4 +386,32 @@ float Timing::dynamicPower(odb::dbInst* inst, sta::Corner* corner)
...
+  sta::dbSta* sta = getSta();
+  sta::dbNetwork* network = sta->getDbNetwork();
+  sta::Cell* cell = network->dbToSta(master);
+  std::vector<odb::dbMaster*> masterSeq;
+  if (cell) {
+    sta::LibertyCell* libcell = network->libertyCell(cell);
+    sta::LibertyCellSeq* equiv_cells = sta->equivCells(libcell);
+    if (equiv_cells) {
+      for (sta::LibertyCell* equiv_cell : *equiv_cells) {
+        odb::dbMaster* master_
```
Comment (maliberty) [inline src/Timing.cc:407]: master_ looks like a member variable.  Perhaps equiv_master?

File: src/Timing.cc [hunk 3]
```diff
@@ -385,4 +386,32 @@ float Timing::dynamicPower(odb::dbInst* inst, sta::Corner* corner)
...
+            = reinterpret_cast<odb::dbMaster*>(equiv_cell->extCell());
```
Comment (maliberty) [inline src/Timing.cc:408]: Use dbNetwork::staToDb for conversion from LibertyCell to dbMaster.

**Review by maliberty — CHANGES_REQUESTED** [review]

File: include/ord/Timing.h (hunk 1 above)
Comment (bingyuew) [reply include/ord/Timing.h]: Fixed that!

File: src/Timing.cc (hunk 2 above)
Comment (bingyuew) [reply src/Timing.cc]: Fixed that!

File: src/Timing.cc (hunk 3 above)
Comment (bingyuew) [reply src/Timing.cc]: Fixed that!

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
Needs a clang-format
